packages/documentation-framework/scripts/webpack/webpack.server.config.js

```diff
@@ -34,6 +34,10 @@ const serverConfig = async (env, argv) => {
           test: /(novnc-core|@novnc\/novnc)\/.*\.js/,
           use: 'null-loader'
         },
+        {
```

Comment:
We probably can back out this null-loader change since the whole directory is going to be ignored by the ssr build.